src/js/utils/StringUtil.js

```diff
@@ -191,6 +191,14 @@ const StringUtil = {
       return null;
     }
 
+    const renderer = new marked.Renderer();
```

Comment:
Would you mind adding some test cases for this? maybe something that could break it like

```md
[foo](http://a.com/<a-mean-uri)
```

Reply:
What would be the ideal behaviour in that case?